src/client/src/core/util/validation/__tests__/rules.spec.ts

```diff
@@ -1073,7 +1073,7 @@ describe('core/util/validation/rules::digitsOnlyWithTwoDecimalPoints', () => {
 
   test('digitsOnlyWithTwoDecimalPoints:: with one decimal points', () => {
     const result = digitsOnlyWithTwoDecimalPoints('123.9');
```

Comment:
is this converted to `123.90` when saving or save `123.9` as it is

Reply:
yes